domain/application/service/migration.go

```diff
@@ -276,28 +277,23 @@ func (s *MigrationService) GetApplicationScaleState(ctx context.Context, name st
 // returning an error satisfying [applicationerrors.ApplicationAlreadyExists]
 // if the application already exists.
 func (s *MigrationService) ImportApplication(ctx context.Context, name string, args ImportApplicationArgs) error {
```

Comment:
This should be ImportIAASApplication and ImportCAASApplication. Then we don't need to check the model in the service.